api/src/main/java/com/codeforcommunity/rest/subrouter/TeamsRouter.java

```diff
@@ -22,12 +28,143 @@ public TeamsRouter(ITeamsProcessor processor) {
   @Override
   public Router initializeRouter(Vertx vertx) {
     Router router = Router.router(vertx);
-
+    registerCreateTeam(router);
+    registerGetTeam(router);
+    registerAddGoal(router);
+    registerDeleteGoal(router);
     registerDisbandTeam(router);
+    registerApplyToTeam(router);
+    registerInviteUser(router);
+    registerApproveUser(router);
+    registerRejectUserRoute(router);
+    registerGetApplicants(router);
+    registerKickUserRoute(router);
+    registerLeaveTeamRoute(router);
+    registerTransferTeamOwnership(router);
 
     return router;
   }
 
+  // Create a Team
+  private void registerCreateTeam(Router router) {
+    Route createTeamRoute = router.post("/create");
+    createTeamRoute.handler(this::handleCreateTeamRoute);
+  }
+
+  private void handleCreateTeamRoute(RoutingContext ctx) {
+    JWTData userData = ctx.get("jwt_data");
+    CreateTeamRequest createTeamRequest =
+        RestFunctions.getJsonBodyAsClass(ctx, CreateTeamRequest.class);
+    processor.createTeam(userData, createTeamRequest);
+    end(ctx.response(), 200);
+  }
```

Comment:
typically in other places, we list all of the register methods, then all of the handler methods.

Reply:
Is there an advantage to this? I personally find this way easier to read, do you prefer the other way?.

Reply:
yeah honestly I like this way better now that I think about it